tests/playwright-tests/.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
```

Comment:
As we perform the rest of our builds and unit test runs in Azure DevOps, it'd be nice to have these incorporated in DevOps too.  I'm wondering what the effort is of creating a YAML pipeline for doing the same as this but in an Azure DevOps-compatible YAML file?

See `azure-pipelines.dfe.yml` and search for `pnpm test:ci` for an example of where we're running unit tests against a TS project.

Not suggesting that we need to do this as part of this PR, just interested to know, and if we adopt Playwright going forwards, Azure DevOps pipelines would be how we'd want to support them.